{
  "id": "oval:com.redhat.cve:def:202324580",
  "version": "636",
  "class": "vulnerability",
  "metadata": {
    "title": "Potential denial-of-service vulnerability in file uploads (moderate)",
    "reference": [
      {
        "ref_id": "CVE-2023-24580",
        "ref_url": "https://access.redhat.com/security/cve/CVE-2023-24580",
        "source": "CVE"
      }
    ],
    "description": "DOCUMENTATION: A memory exhaustion flaw was found in the python-django package. This issue occurs when passing certain inputs, leading to a system crash and denial of service.",
    "advisory": {
      "from": "secalert@redhat.com",
      "severity": "Moderate",
      "updated": {
        "date": "2023-08-21"
      },
      "cve": [
        {
          "text": "CVE-2023-24580",
          "cvss3": "7.5/CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
          "cwe": "CWE-400",
          "href": "https://access.redhat.com/security/cve/CVE-2023-24580",
          "impact": "moderate",
          "public": "20230214:0900"
        }
      ],
      "affected": {
        "resolution": [
          {
            "state": "Out of support scope",
            "component": [
              "python-django",
              "python-django-bash-completion"
            ]
          }
        ]
      },
      "affected_cpe_list": {
        "cpe": [
          "cpe:/a:redhat:ceph_storage:3",
          "cpe:/a:redhat:ceph_storage:3::el7"
        ]
      },
      "issued": {}
    },
    "affected": {}
  },
  "criteria": {
    "operator": "OR",
    "criterias": [
      {
        "operator": "AND",
        "criterias": [
          {
            "operator": "OR",
            "criterias": [
              {
                "operator": "AND",
                "criterions": [
                  {
                    "test_ref": "oval:com.redhat.cve:tst:201814574003",
                    "comment": "python-django is installed"
                  },
                  {
                    "test_ref": "oval:com.redhat.cve:tst:201814574004",
                    "comment": "python-django is signed with Red Hat redhatrelease2 key"
                  }
                ]
              },
              {
                "operator": "AND",
                "criterions": [
                  {
                    "test_ref": "oval:com.redhat.cve:tst:201814574001",
                    "comment": "python-django-bash-completion is installed"
                  },
                  {
                    "test_ref": "oval:com.redhat.cve:tst:201814574002",
                    "comment": "python-django-bash-completion is signed with Red Hat redhatrelease2 key"
                  }
                ]
              }
            ]
          }
        ],
        "criterions": [
          {
            "test_ref": "oval:com.redhat.cve:tst:201610745003",
            "comment": "Red Hat Enterprise Linux 7 is installed"
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:com.redhat.cve:tst:201610745004",
        "comment": "Red Hat Enterprise Linux must be installed"
      }
    ]
  }
}
